docs/python.md

```diff
@@ -1,3 +1,33 @@
 # Awesome Open Source - Python
 
-Coming Soon!
+- [APScheduler](https://github.com/agronholm/apscheduler) - A lightweight, in-process task scheduler.
+- [Arrow](https://github.com/arrow-py/arrow) - Better dates & times for Python.
+- [Bokeh](https://github.com/bokeh/bokeh) - Interactive web plotting for Python.
+- [Crossbar](https://github.com/crossbario/crossbar/) - Open-source Unified Application Router (Websocket & WAMP for Python on Autobahn).
+- [Datasette](https://github.com/simonw/datasette) - An open source multi-tool for exploring and publishing data.
+- [dejavu](https://github.com/worldveil/dejavu) - Audio fingerprinting and recognition.
+- [djedi-cms](http://djedi-cms.org/) - A lightweight but yet powerful Django CMS with plugins, inline editing and performance in mind.
+- [Dramatiq](https://github.com/Bogdanp/dramatiq) - A fast and reliable background task processing library for Python 3.
+- [FastAPI Users](https://github.com/fastapi-users/fastapi-users) - Ready-to-use and customizable users management for FastAPI.
+- [Faker](https://github.com/joke2k/faker) - Faker is a Python package that generates fake data for you.
+- [fuzzywuzzy](https://github.com/seatgeek/fuzzywuzzy) - Fuzzy String Matching.
+- [HTTPie](https://github.com/httpie/httpie) - Modern command line HTTP client, user-friendly curl alternative.
+- [mrq](https://github.com/pricingassistant/mrq) - Mr. Queue - A distributed worker task queue in Python using Redis & gevent.
+- [p](https://github.com/qw3rtman/p) - Dead simple interactive Python version management.
+- [PeachPy](https://github.com/Maratyszcza/PeachPy) - x86-64 assembler embedded in Python. Can be used as inline assembler for Python or as a stand-alone assembler for Windows, Linux, OS X, Native Client and Go.
+- [Pendulum](https://github.com/sdispater/pendulum) - Python datetimes made easy.
+- [Peewee](https://github.com/coleifer/peewee) - A small, expressive ORM.
+- [Poetry](https://python-poetry.org/) - Python dependency management and packaging made easy.
+- [Pony ORM](https://github.com/ponyorm/pony) - ORM that provides a generator-oriented interface to write database queries.
+- [Pydantic](https://github.com/pydantic/pydantic) - Data validation and settings management using Python type annotations.
+- [pyocr](https://github.com/jflesch/pyocr) - A wrapper for Tesseract and Cuneiform.
```

Comment:
this project has moved to: https://gitlab.gnome.org/World/OpenPaperwork/pyocr

could you please update this?

Reply:
Its not as frequently updated, so removed in favor of pytesseract.